modules/ztp-add-local-reg-for-sno-duprofile.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: CONCEPT
+[id="ztp-add-local-reg-for-sno-duprofile_{context}"]
+= Configuring the Image Registry Operator for local caching of images
+
+{product-title} manages image caching with a local registry. A Single Node OpenShift (SNO) Distributed Unit (DU) is subject to bandwidth restrictions when communicating with the centralized registry. This can lead to long image download times.
+
+This is unavoidable for the initial deployment. Over time, there is a risk that `crio` will wipe `/var/lib/containers/storage` in the case of a disorderly shutdown.
+
+To address this, you need a local image registry on the SNO. The cluster images need to be  mirrored to this local image registry along with being used as the registry for the applications. This is useful in Edge scenarios where an SNO is deployed on the Far Edge. The SNO cannot access the internet, just its own resources and eventually the internal network.
```

Comment:
I think we should remove `The cluster images need to be mirrored to this local image registry`  since it's in contradiction with the NOTE section.

Reply:
Done.